care/utils/assetintegration/base.py

```diff
@@ -16,6 +16,7 @@ def __init__(self, meta):
         self.host = self.meta["local_ip_address"]
         self.middleware_hostname = self.meta["middleware_hostname"]
         self.insecure_connection = self.meta.get("insecure_connection", False)
+        self.timeout = getattr(settings, "MIDDLEWARE_API_TIMEOUT", 20)
```

Comment:
this is not required 
you can reference it directly from settings.MIDDLEWARE_REQUEST_TIMEOUT